CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change log
 
 ## master (unreleased)
+* [#2927](https://github.com/bbatsov/rubocop/issues/2927): Add autocorrect for `Rails/Validation` cop. ([@neodelf][])
```

Comment:
This should be under `### New features`, which you'll have to add as well. Also - we also add blank lines between section titles and the items in the section.